mpags-cipher.cpp

```diff
@@ -0,0 +1,50 @@
+#include <vector>
+#include <string.h>
+#include <iostream>
+#include <stdio.h>
```

Comment:
```suggestion
#include <vector>
#include <string>
#include <iostream>
```
You don't need the two C headers you have included, while you do need the C++ header for the string class.
There are situations where you may need certain C functions, e.g. `isalpha`, and then you should use the C++ version of the header, which places those functions in the `std` namespace rather than in the global namespace. In the case of `isalpha`, etc. the header you need is `<cctype>`.